lighthouse-core/config/default.js

```diff
@@ -292,6 +306,16 @@ module.exports = {
         {id: 'valid-lang', weight: 1, group: 'a11y-language'},
         {id: 'video-caption', weight: 4, group: 'a11y-describe-contents'},
         {id: 'video-description', weight: 3, group: 'a11y-describe-contents'},
+        {id: 'custom-controls-labels', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'custom-controls-roles', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'focus-traps', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'focusable-controls', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'heading-levels', weight: 0, group: 'manual-a11y-checks'},
```

Comment:
is the order of these deliberate?

seems like its mostly: ARIA, Focus, Order and Exclusion

i dont really want to add groupings inside of Manual, but i figure they should be mostly grouped by topic. :)

Reply:
good call it should be. They're split amongst keyboard, screen reader, and landmarks but I think i may have just alphabetized them cuz my brain was shutting down. I can resort